external/fv3fit/examples/append_halo.py

```diff
@@ -0,0 +1,25 @@
+import xarray as xr
```

Comment:
I made this to test out the halo update code, could delete it though I thought I'd propose adding it as an example. Feels kind of strange as an example since this is private API within fv3fit, so I could go either way.